packages/expo-video/android/src/main/java/expo/modules/video/AdManagerBuilder.kt

```diff
@@ -0,0 +1,30 @@
+package expo.modules.video
+
+import android.content.Context
+import android.util.Log
+import expo.modules.kotlin.AppContext
+import expo.modules.video.interfaces.AdManager
+import expo.modules.video.interfaces.stubs.AdManagerStub
+
+private const val LOG_TAG = "AdManagerFactory"
+
+// reflection-based dynamic loading of IMA plugin
+object AdManagerFactory {
+  fun create(context: Context, appContext: AppContext?): AdManager {
+    if (BuildConfig.useInteractiveMediaAds) {
```

Comment:
Nit: I would use an early exit here, that should help readability. Makes it easier to understand how this code works.